packages/govuk-frontend/src/govuk-prototype-kit/functions.js

```diff
@@ -0,0 +1,4 @@
+const { addFunction } = require('govuk-prototype-kit').views
+const config = require('govuk-prototype-kit/lib/config.js').getConfig(null)
+
+addFunction('govukRebrand', () => config.rebrand ?? false)
```

Comment:
@owenatgov A late thought as I was thinking of documentation, could we prefix or namespace the variable we expect inside the kit's config to avoid collisions? Not sure if there's precedents in other plugins, but we could at least do `govukRebrand` if not `govukFrontendRebrand`, unless the kit has some convention like `config['package-name-of-the-plugin'].propertyName`?

Reply:
I think that's fair! I'm not aware of any convention but I can dig around before making the change.

Reply:
Thought about possible conventions some more and I don't think there would be one... as the `config` part of the Prototype Kit is not publicly documented. 

I'd be keen to err to something as verbose as possible for future proofness, so possibly `config.plugins['name-of-the-plugin-package'].propertyName` which would isolate:
- plugins configuration from the kit's configuration itself
- between plugins, each plugin's configuration in its own namespace

Which in our case would give `config.plugins['govuk-frontend'].rebrand` as a property to read in the config. We'd need to check for the support for the `?.` operator in the kit based on the versions of Node it can run on... which makes me think we need to check for `??` as well 😬

Reply:
In your proposition, would users need to add the following to their prototypes?

```json
{
  "govuk-frontend": {
    "rebrand": true
  }
}
```